packages/playwright/tests/chromedash-guide-new-page_pwtest.js

```diff
@@ -18,20 +19,30 @@ test.afterEach(async ({ page }) => {
 
 test('navigate to create feature page', async ({page}) => {
   // console.log('navigate to create feature page');
+  const mobile = await isMobile(page);
+
+  const menuButton = page.locator('[data-testid=menu]');
+  await expect(menuButton).toBeVisible();
+  if (mobile) {
+    await menuButton.click();  // To show menu.
+  }
 
   // Expect create feature button to be present.
   const createFeatureButton = page.getByTestId('create-feature-button');
   await expect(createFeatureButton).toBeVisible({timeout: 30000});
+  // Take a screenshot of the "Create feature" button.
+  await expect(createFeatureButton).toHaveScreenshot('create-feature-button.png');
 
-  // Take a screenshot of header with "Create feature" button.
-  await expect(page.getByTestId('header')).toHaveScreenshot('create-feature-button.png');
-
-  // Navigate to the new feature page by clicking.
-  createFeatureButton.click();
+  // Navigate to the new feature page.
+  await createFeatureButton.click();
+  if (mobile) {
+    await menuButton.click();  // To hide menu
+    delay(500);
+  }
 
   // Expect "Add a feature" header to be present.
   const addAFeatureHeader = page.getByTestId('add-a-feature');
-  await expect(addAFeatureHeader).toBeVisible();
+  await expect(addAFeatureHeader).toBeVisible({timeout:10000});
```

Comment:
Why so many different timeout values at different places in the tests?  Would it be better to decide on a standard test step timeout?  Or maybe one for steps that require no API calls and a longer one for steps that do require API calls.

Reply:
The varying timeouts make sense in some cases, since it takes a varying amount of time before some displays stabilize.   The default is 5000, but that often seems to not be enough.  Changing the default to, say, 10000 might be reasonable.    It seems to be required for this step, for example.

Early on, many steps were timing out, and I couldn't figure out why, and the turnaround for trying something different became very tedious.  Now that things have stabilized, I'm trying to incrementally reduce timeouts and eliminate delays.